content/sensu-go/6.2/operations/control-access/create-limited-service-accounts.md

```diff
@@ -0,0 +1,332 @@
+
+{{< code shell >}}
+sensuctl asset add sensu/sensu-ec2-handler:0.4.0
+{{< /code >}}
+
+The response will indicate that the asset was added:
+
+{{< code shell >}}
+fetching bonsai asset: sensu/sensu-ec2-handler:0.4.0
+added asset: sensu/sensu-ec2-handler:0.4.0
+
+You have successfully added the Sensu asset resource, but the asset will not get downloaded until
+it's invoked by another Sensu resource (ex. check). To add this runtime asset to the appropriate
+resource, populate the "runtime_assets" field with ["sensu/sensu-ec2-handler"].
+{{< /code >}}
+
+You can also download the dynamic runtime asset definition for Debian or Alpine from [Bonsai][13] and register the asset with `sensuctl create --file filename.yml`.
+
+{{% notice note %}}
+**NOTE**: Sensu does not download and install dynamic runtime asset builds onto the system until they are needed for command execution.
+Read [the asset reference](../../../plugins/assets#dynamic-runtime-asset-builds) for more information about dynamic runtime asset builds.
+{{% /notice %}}
+
+## Configure an EC2 handler for the service account
+
+To configure the EC2 handler, you will need AWS account credentials and details for the AWS instance you want to manage, like the AWS instance ID.
+You will also need the API key for the `ec2-service` user.
+
+{{% notice note %}}
+**NOTE**: Use [secrets management](../../manage-secrets/secrets-management/) to configure environment variables for your AWS access and secret keys and the `ec2-service` user's API key and avoid exposing this sensitive information by listing it directly in the EC2 handler definition.<br><br>
+The [Sensu Go EC2 Handler's Bonsai page](https://bonsai.sensu.io/assets/sensu/sensu-ec2-handler#environment-variables) includes an example for configuring secrets definitions with Sensu's built-in [`env` secrets provider](../../manage-secrets/secrets-providers/#env-secrets-provider-example).
+{{% /notice %}}
+
+In the following code, replace the placeholders for these attributes with valid values:
+
+- `aws-region`: the AWS region where your EC2 instance is located.
+- `aws-instance-id-label`: the Sensu entity label that contains the AWS instance ID.
+If your AWS EC2 instance entities do not include labels that specify the instance ID, use `aws-instance-id` instead and specify the AWS instance IDs themselves.
+- `sensu-api-url`: the Sensu API URL.
+
+Then, run this command with your valid values in place to create the handler defintion:
+
+{{< language-toggle >}}
+
+{{< code text "YML" >}}
+cat << EOF | sensuctl create
+---
+type: Handler
+api_version: core/v2
+metadata:
+  namespace: default
+  name: sensu-ec2-handler
+spec:
+  type: pipe
+  runtime_assets:
+    - sensu/sensu-ec2-handler
+  filters:
+    - is_incident
+    - not_silenced
+  command: >-
+    sensu-ec2-handler
+    --aws-region region
+    --aws-instance-id-label aws-instance-id
+    --aws-allowed-instance-states pending,running
+    --sensu-api-url http://localhost:8080
```

Comment:
Should this be the complete entities API URL?

Reply:
No, just the base of the URL is fine the remainder gets constructed based on the event.